group-actors/src/main/java/org/sunbird/models/GroupResponse.java

```diff
@@ -0,0 +1,26 @@
+package org.sunbird.models;
```

Comment:
it seems memberRole is 1:n relationship to members?

Reply:
A member can be either admin or member in group, so a memberRole will a user is either admin or member in a particular group